ja/_posts/2017-12-29-groonga-7.1.0.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: post.ja
+title: Groonga 7.1.0リリース
+description: Groonga 7.1.0をリリースしました！
+published: false
+---
+
+## Groonga 7.1.0リリース
+
+今日は肉の日ですね！
+
+[Groonga 7.1.0](/ja/docs/news.html#release-7.1.0)をリリースしました！
+
+それぞれの環境毎のインストール方法: [インストール](/ja/docs/install.html)
+
+### 変更内容
+
+主な変更点は以下の通りです。
+
+  * [load](/ja/docs/reference/commands/load.html) のクエリーログのフォーマットを改良しました。
+
+  * [logical_count](/ja/docs/reference/commands/logical_count.html) のクエリーログのフォーマットを改良しました。
+
+  * [logical_select](/ja/docs/reference/commands/logical_select.html) のクエリーログのフォーマットを改良しました。
+
+  * [delete](/ja/docs/reference/commands/delete.html) のクエリーログのフォーマットを改良しました。
+
+  * ドリルダウンでベクター型のカラムをサポートしました。
+
+  * `grn_bulk_*()` APIにおいて、 `realloc()` が呼ばれる回数を削減しました。
+
+  * 関数 `index_column_source_records` を新規に追加しました。インデックスカラムのソースを取得することができます。
+
+### [load](/ja/docs/reference/commands/load.html) のクエリーログのフォーマットを改良しました。
+
+[load](/ja/docs/reference/commands/load.html)のクエリーログに以下の情報が表示されるようになります。
+
+  * ロードしたレコード数
+  * ロードがエラーになったレコードとカラムの数
+  * 総レコード数
+
+具体的には、以下のように表示されます。
+
+```text
+#:000000000000000 load(3): [1][2][3]
+#<000000000000000 rc=-22
```

Comment:
We should use real line in query log instead of grntest output.
Because users see query log not grntest output.

Reply:
You can use `rm -rf /tmp/db && mkdir -p /tmp/db && groonga --query-log-path /tmp/db/query.log --file test/command/suite/... -n /tmp/db/db` to get real line in query log.